unison-src/transcripts/help.md

```diff
@@ -0,0 +1,6 @@
+```ucm
+.> help push
+.> help
+.> ?
+.> help-topics
```

Comment:
all valid scenarios covered (that I am aware of at least; please comment if I am missing something.)